docs/source/guide/benchmarks.md

```diff
@@ -0,0 +1,194 @@
+
+The {func}`.generate_mirror_circuit` involves running a quantum circuit forward and then “mirroring” it (applying the reverse operations). Ideally, this results in returning the system to the initial state, so they’re great for testing if the noise mitigation is effective in preserving information through complex sequences.
+
+```{code-cell} ipython3
+from typing import List, Tuple
+import numpy as np
+import cirq
+import networkx as nx
+from mitiq import zne
+from mitiq import benchmarks
+
+topology = nx.complete_graph(3)
+
+def get_circuit(depth: int) -> Tuple[cirq.Circuit, List[int]]:
+    circuit, correct_bitstring = benchmarks.generate_mirror_circuit(
+        nlayers=depth,
+        two_qubit_gate_prob=1.0,
+        connectivity_graph=topology,
+        return_type="cirq",
+    )
+    return circuit, correct_bitstring
+
+circuit, correct_bitstring = get_circuit(depth=3)
+
+def execute(
+    circuit: cirq.Circuit,
+    correct_bitstring: List[int],
+    noise_level=0.005,
+) -> float:
+    """Executes the input circuit(s) and returns ⟨A⟩, where
+    A = |correct_bitstring⟩⟨correct_bitstring| for each circuit.
+    """
+    noisy_circuit = circuit.with_noise(cirq.depolarize(p=noise_level))
+
+    noisy_circuit += cirq.measure(*sorted(circuit.all_qubits()), key="m")
+    backend = cirq.DensityMatrixSimulator()
+
+    backend = cirq.DensityMatrixSimulator()
+
+    result = backend.run(noisy_circuit)
+    expval = result.measurements["m"].tolist().count(correct_bitstring)
+    return expval
+
+def execute_with_fixed_bitstring(circuit, noise_level=0.005):
+    return execute(circuit, correct_bitstring, noise_level)
+
+print(circuit)
+print(circuit.final_state_vector()) # Shows the ideal circuit state
+
+true_value = execute(circuit, correct_bitstring,noise_level=0.0)
+noisy_value = execute(circuit, correct_bitstring)
+
+zne_value = zne.execute_with_zne(circuit, execute_with_fixed_bitstring)  # Noisy quantum computer + Mitiq
+
+print(f"Error w/o  Mitiq: {abs((true_value - noisy_value) / true_value):.3f}")
+print(f"Error w Mitiq:    {abs((true_value - zne_value) / true_value):.3f}")
+```
+
+## Mirror Quantum Volume Circuits
+
+The {func}`.generate_mirror_qv_circuit` is designed to test `quantum volume`, a metric combining circuit depth, number of qubits, and fidelity. These circuits check whether error mitigation techniques help achieve higher effective quantum volumes on noisy devices.
+
+```{code-cell} ipython3
+from mitiq.benchmarks import generate_mirror_qv_circuit
+
+circuit= generate_mirror_qv_circuit(num_qubits=7, depth=2)
+print(circuit)
+```
+
+## Quantum Phase Estimation Circuits
+
+The {func}`.generate_qpe_circuit` is used to the measure eigenvalues of unitary operators. Since accurate phase estimation requires precise control over operations, these circuits test the mitigation techniques’ ability to handle small noise effects over multiple gate sequences.
+
+```{code-cell} ipython3
+from mitiq.benchmarks import generate_w_circuit
```

Comment:
wrong function? 🤔

Reply:
Yes. Sorry about this one. Don't know how this happened. Will correct this in the next commit.